packages/react-components/react-tooltip-v2-preview/library/src/components/Tooltip/useTooltipStyles.styles.ts

```diff
@@ -0,0 +1,66 @@
+import { makeStyles, mergeClasses } from '@griffel/react';
+import { createArrowStyles } from '@fluentui/react-positioning';
+import { tokens } from '@fluentui/react-theme';
+import { arrowHeight } from './private/constants';
+import type { TooltipSlots, TooltipState } from './Tooltip.types';
+import type { SlotClassNames } from '@fluentui/react-utilities';
+
+export const tooltipClassNames: SlotClassNames<TooltipSlots> = {
+  content: 'fui-Tooltip__content',
+};
+
+/**
+ * Styles for the tooltip
+ */
+const useStyles = makeStyles({
+  root: {
+    boxSizing: 'border-box',
+    maxWidth: '240px',
+    cursor: 'default',
+    fontFamily: tokens.fontFamilyBase,
+    fontSize: tokens.fontSizeBase200,
+    lineHeight: tokens.lineHeightBase200,
+    overflowWrap: 'break-word',
+    borderRadius: tokens.borderRadiusMedium,
+    border: `1px solid ${tokens.colorTransparentStroke}`,
+    padding: '4px 11px 6px 11px', // '5px 12px 7px 12px' minus the border width '1px'
+    backgroundColor: tokens.colorNeutralBackground1,
+    color: tokens.colorNeutralForeground1,
+
+    // TODO need to add versions of tokens.alias.shadow.shadow8, etc. that work with filter
+    filter:
+      `drop-shadow(0 0 2px ${tokens.colorNeutralShadowAmbient}) ` +
+      `drop-shadow(0 4px 8px ${tokens.colorNeutralShadowKey})`,
+
+    margin: 0,
+    inset: 'auto',
+    width: 'max-content',
+  },
+
+  inverted: {
+    backgroundColor: tokens.colorNeutralBackgroundStatic,
+    color: tokens.colorNeutralForegroundStaticInverted,
+  },
+
+  arrow: createArrowStyles({ arrowHeight }),
```

Comment:
Have you been able to get this working? I struggled with adding it and maintaining the existing DOM structure when prototyping this.